predicators/ml_models.py

```diff
@@ -1354,6 +1354,11 @@ def __init__(self,
         self._num_ground_nsrts = 0
         self._replay_buffer: Deque[MapleQData] = deque(
             maxlen=self._replay_buffer_max_size)
+        self._epsilon = 0.5
+        self._min_epsilon = 0.05
```

Comment:
it's useful to have this, but instead of hard-coding this here, you should make these command-line flags by using `CFG`.